Solutions/Box/Analytic Rules/BoxItemSharedToExternalUser.yaml

```diff
@@ -22,11 +22,10 @@ query: |
   | extend corp_domain = tolower(extract(@'@(.*)', 1, SrcUserUpn))
   | extend accessibleby_domain = tolower(extract(@'@(.*)', 1, AccessibleByLogin))
   | where corp_domain != accessibleby_domain

```

Comment:
same comment above.. Add output formatting, aggregation and additional entitytypes along with UPNDomain